### PR TITLE
chore(dependencies): set pluginPackages for all plugins with pluginId

### PR DESCRIPTION
**Pull request by rgolangh**
Signed-off-by: Roy Golan <rgolan@redhat.com>

[FLPATH-1521](https://issues.redhat.com//browse/FLPATH-1521)

### CODE REVIEW

**Comment by rgolangh** [comment]
@nickboldt please take a look

**Review by ydayagi — DISMISSED** [review]

File: plugins/matomo-backend/package.json [hunk 1]
```diff
@@ -13,7 +13,12 @@
   "backstage": {
     "role": "backend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "matomo"
+    "pluginId": "matomo",
+    "pluginPackages": [
+      "@janus-idp/backstage-plugin-matomo",
```
Comment (nickboldt) [inline plugins/matomo-backend/package.json:18]: so the backend includes the common and front end plugins too?

File: plugins/analytics-module-matomo/package.json [hunk 2]
```diff
@@ -12,7 +12,12 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "analytics-module-matomo"
+    "pluginId": "analytics-module-matomo",
+    "pluginPackages": [
+      "@janus-idp/backstage-plugin-analytics",
```
Comment (nickboldt) [inline plugins/analytics-module-matomo/package.json:17]: this don't depend on / include the matomo backend and common plugins, only the analytics plugin?

File: plugins/kiali-backend/package.json [hunk 3]
```diff
@@ -12,7 +12,12 @@
   "backstage": {
     "role": "backend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "kiali"
+    "pluginId": "kiali",
+    "pluginPackages": [
+      "@janus-idp/backstage-plugin-kiali",
+      "@janus-idp/backstage-plugin-kiali-common",
```
Comment (nickboldt) [inline plugins/kiali-backend/package.json:18]: where does the kiali-common plugin come from?

**Review by nickboldt — CHANGES_REQUESTED** [review]
can't merge as is, have questions! 

alternative option in https://github.com/janus-idp/backstage-plugins/pull/1921 -- not saying it's better, just simpler.

**Comment by rgolangh** [comment]
closing in favour of #1921

**Pull request closed without merging**